intl/messages/test.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
It seems strange to have this file here. Can’t we mock it?

Reply:
Yes, makes more sense, done!

Reply:
@ritazoliveira I think you forgot to remove this file.